Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -718,6 +718,14 @@ def __init__(self, *args, **kwargs):
             kwargs['sigtermTime'] = 10
         super().__init__(*args, **kwargs)
 
+    def buildCommandKwargs(self, warnings):
```

Comment:
Do we need this method? Can we do this in __init__?

Reply:
Unfortunately build properties cannot be accessed in `init`